test/views/MovementsViewSpec.scala

```diff
@@ -89,5 +91,30 @@ class MovementsViewSpec extends WordSpec with MustMatchers with Stubs with ViewV
       getElementById(pageWithData, s"dateOfUpdate-$conversationId").text() must be("2019-10-31 00:10")
       getElementById(pageWithData, s"noOfNotifications-$conversationId").text() must be("1")
     }
+
+    "contain MUCR and DUCR if Submission contains both" in {
+      val pageWithData: Html = new movements(mainTemplate)(
+        Seq(
+          (
```

Comment:
Is this is for tuple ? Maybe `->` would looks better?

Reply:
DONE